packages/core/admin/admin/src/pages/InstalledPluginsPage/Plugins.js

```diff
@@ -55,7 +55,7 @@ const Plugins = () => {
           })}
         />
         <ContentLayout>
-          <Table colCount={2} rowCount={data?.plugins.length + 1}>
+          <Table colCount={2} rowCount={data?.plugins.length || 0 + 1}>
```

Comment:
the previous code was unsafe if plugins doesn't have a length property and could fail the || just fixes that. If you have better happy to change it

Reply:
I see. What about `data?.plugins?.length ?? 0`? Because it only has an optional check, we could assume that `plugins` is always an array, if it is set.

Reply:
I'm fine with that